src/styles/reviews.scss

```diff
@@ -0,0 +1,39 @@
+.card__reviews {
+  margin-top: 7px;
+  align-items: center;
+  text-align: center;
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+
+  &-stars-4 {
+    display: flex;
+  }
+}
+
+.card__stars-star{
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  background-image: url(../images/star.svg);
+
+  &:not(:last-child) {
+    margin-right: 4px;
+  }
+}
+
+.card__reviews-stars-4 .card__stars-star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
+
+.card__review-tex {
+  padding-top: 1px;
```

Comment:
you must have the stars file with the separate stars block and move the card__review to card.scss it's not the separate block


don't hesitate to ask something in the chat

Reply:
corrected